subprojects/docs/src/samples/testing/jacoco/quickstart/kotlin/build.gradle.kts

```diff
@@ -43,7 +43,7 @@ tasks.jacocoTestReport {
 // end::report-configuration[]
 
 // tag::violation-rules-configuration[]
-tasks.jacocoTestCoverageVerification {
+tasks.named<JacocoCoverageVerification>("jacocoTestCoverageVerification") {
```

Comment:
:x: `tasks. jacocoTestCoverageVerification {` is already lazy

Reply:
Revert is pending.